### PR TITLE
FOUR-10511 modeler integration create/remove elements

### PR DESCRIPTION
**Pull request by devmiguelangel**
## Issue & Reproduction Steps

Expected behavior:
Integrate the functionality of creating and removing elements between the Modeler and the Microservice realtime

Actual behavior: 
n/a

## How to Test
1. Run the microservice realtime server under the [FOUR-10511](https://github.com/ProcessMaker/microservice-realtime/tree/feature/FOUR-10511) branch
2. Run the Modeler
3. Set the `VUE_APP_WEBSOCKET_PROVIDER` environment variable to `ws://127.0.0.1:3000`
4. Create and remove elements through two browsers

## Related Tickets & Packages

[FOUR-10511](https://processmaker.atlassian.net/browse/FOUR-10511)
[FOUR-10504](https://processmaker.atlassian.net/browse/FOUR-10504)
[FOUR-10505](https://processmaker.atlassian.net/browse/FOUR-10505)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

[FOUR-10511]: https://processmaker.atlassian.net/browse/FOUR-10511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-10511]: https://processmaker.atlassian.net/browse/FOUR-10511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-10504]: https://processmaker.atlassian.net/browse/FOUR-10504?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-10505]: https://processmaker.atlassian.net/browse/FOUR-10505?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/multiplayer/multiplayer.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    // Get the node id generator
+    this.#nodeIdGenerator = getNodeIdGenerator(this.modeler.definitions);
+    // Get the room name from the process id
+    this.room = new Room(`room-${window.ProcessMaker.modeler.process.id}`);
+
+    // Connect to websocket server
+    this.clientIO = io(process.env.VUE_APP_WEBSOCKET_PROVIDER, { transports: ['websocket', 'polling']});
+
+    this.clientIO.on('connect', () => {
+      // console.log('########################');
```
Comment (danloa) [inline src/multiplayer/multiplayer.js:30]: Remove the commented code

File: src/setup/globals.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ window.ProcessMaker = {
   },
   modeler: {
     process: {
-      id: 1,
+      id: 4,
```
Comment (danloa) [inline src/setup/globals.js:54]: why this was changed to 4?

File: src/multiplayer/multiplayer.js (hunk 1 above)
Comment (devmiguelangel) [reply src/multiplayer/multiplayer.js]: Comments were removed

File: src/setup/globals.js (hunk 2 above)
Comment (devmiguelangel) [reply src/setup/globals.js]: It was for testing purposes, It has been restored

**Review by danloa — APPROVED** [review]

**Pull request merged**